Document hotfix process (#701)
openstack-k8s-operators/dataplane-operator: docs/assemblies/hotfixing-the-data-plane.adoc

[id="assembly_hotfixing-the-data-plane"]
= Hotfixing the data plane

:context: dataplane

[role="_abstract"]

> The OpenStack DataPlane can be updated when hotfix content is available. Hotfix content

Reviewer: ```suggestion
You can update the OpenStack data plane when hotfix content is available. Hotfix content
```
Use active voice where possible.

can be delivered as RPM packages or container images. How the software is
installed on the data plane nodes determines how the hotfix content needs to be
applied.

> include::proc_hotfixing-the-data-plane-rpm-content.adoc[leveloffset=+1]

Reviewer: This include needs to be moved to after all the text in the assembly. Can you move it to just before the  include on line 19?

Author: Updated with suggestion

> Container hotfix content needs to be applied to the data plane nodes such that
> any running containers are updated to run from container images where the
> hotfix content has been applied. Container hotfix content could be delivered as
> either RPM's or already updated container images.

Reviewer: ```suggestion
You apply a container hotfix to the data plane nodes by updating any running containers to run from container images where the hotfix content has been applied. Container hotfix content can be delivered as
either RPM's or already updated container images.
```
